### DISCUSSION TITLE
"Уплывание" еженедельного корабля за горизонт

### DISCUSSION THREAD

**Discussion by BasilMas** [fedorovvl/tso_client#408] [General]
При манипулировании размером окна во время загрузки кораблик (и сейчас в ивент голем тоже) уплывает за левый верхний угол (см скрин)
![image](https://github.com/user-attachments/assets/a3d9d261-cb3f-4d41-81fd-b720abaccd40)

Catavasiya — 27.10.2024 20:07
у меня ощущение, что это бывает, когда я толком не дождавшись загрузки острова, принудительно изменяю размер окна.
т.е. кейс примерно такой: я хочу стартануть основной акк и мульта в максимальных окнах. поэтому "maximized" у меня стоит у обоих в настройках запуска. 
стартую основной акк и, не дождавшись появления острова, минимизирую окно, тыкая в правый нижний угол панели задач винды, чтобы стал виден рабочий стол с ярлыком запуска мульта. 
пока более точной диагностики этого улезания окна еженедельки/голема за окно клиента у меня нет. 

Лечится перезапуском 

Как воспроизвести?
- запустить клиента с параметром --window maximized
- пока идет загрузка - бежит ползунок на заставке, свернуть окно с клиентом
- дождаться, пока все загрузится
- вызвать еженедельный кораблик/голема
- удивиться

Вопрос - можно ли пофиксить, или принять как должное, понять и простить
Спасибо!

**Comment by fedorovvl** [comment]
не манипулируйте размером окна во время загрузки..

**Comment by fedorovvl** [comment]
```
addToolsMenuItem("Fix Golem window", function() { swmmo.application.GAMESTATE_ID_EVENT_MONSTER.y = swmmo.application.GAMESTATE_ID_EVENT_MONSTER.x = 0; });
```
фикс

**Comment by BasilMas** [comment]
Работает, спасибо! Можно в дискуссию перенести?
